.gitignore

```
@@ -1,16 +1,37 @@
-# system files
-*.log
-.DS_Store*
+## editors
+/.idea
+
+## system files
+.DS_Store
 .idea/
+*.log
+
```

As VSCode becomes more and more popular IDE, I'd recommend adding its files to the `.gitignore` list
```suggestion
.vscode/*
!.vscode/settings.json
!.vscode/tasks.json
!.vscode/launch.json
!.vscode/extensions.json
```